README.md

```diff
@@ -151,7 +151,10 @@ Vertical or horizontal lines are supported.
         content: "Test label",
 
         // Rotation of label, in degrees, default is 0
-        rotation: 90
+        rotation: 90,
```

Comment:
How about making the `rotation` accept `'auto'` instead of adding another option and needing complex if statements to figure out what to do (and ending up ignoring the rotation option)?

Reply:
> 
> 
> How about making the `rotation` accept `'auto'` instead of adding another option and needing complex if statements to figure out what to do (and ending up ignoring the rotation option)?

You're right!!! I really didn't think to implement in this way... I'm gonna change it as suggested